astroid/rebuilder.py

```diff
@@ -106,7 +107,7 @@ def __init__(
     def _get_doc(self, node: T_Doc) -> Tuple[T_Doc, Optional[str]]:
         try:
             if PY37_PLUS and hasattr(node, "docstring"):
-                doc = node.docstring
+                doc = node.docstring  # type: ignore # mypy doesn't recognize hasattr
```

Comment:
```suggestion
                doc = node.docstring  # type: ignore[attr-defined]  # mypy doesn't recognize hasattr
```
https://github.com/PyCQA/astroid/pull/1247#discussion_r750803599